examples/with-next-sass/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "sass-next",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "author": "",
```

Comment:
Remove this.

Reply:
Just a quick question on this one, consider that we have added the `/_next/static/style.css` to the app to make it compatible with the production build, should we remove the `build` and `start` command from the `package.json`?

Reply:
I referring to the author key